src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -611,11 +622,12 @@ public static List<String> getFallbackConnectionURL(
     }
 
     private boolean localCheckoutProvided() {
-        return config.getLocalCheckoutDir() != null && config.getLocalCheckoutDir().exists();
+        File localCheckoutDir = config.getLocalCheckoutDir();
+        return localCheckoutDir != null && localCheckoutDir.exists();
     }
 
     private boolean onlyOnePluginIncluded() {
-        return config.getIncludePlugins() != null && config.getIncludePlugins().size() == 1;
+        return config.getIncludePlugins().size() == 1;
```

Comment:
This null check is now redundant due to the way we initialize the list with arguments from picocli.